src/Components/Fsi.fs

```diff
@@ -559,7 +580,13 @@ module Fsi =
 
                 let text = editor.document.getText range
 
-                do! send terminal text
+                // Skip comments
+                let textWithoutComments =
```

Comment:
skip comments in `send fsi` many times comments can breach window size of editor, resulting in known bug/issue